Ginger/GingerCoreNET/Drivers/CoreDrivers/Web/Selenium/SeleniumDriver.cs

```diff
@@ -9209,15 +9209,15 @@ public string GetCurrentPageSourceString()
 
         public void SetCurrentPageLoadStrategy(DriverOptions options)
         {
-            if (PageLoadStrategy.ToLower() == nameof(OpenQA.Selenium.PageLoadStrategy.Normal).ToLower())
+            if (PageLoadStrategy!= null && PageLoadStrategy.ToLower() == nameof(OpenQA.Selenium.PageLoadStrategy.Normal).ToLower())
             {
                 options.PageLoadStrategy = OpenQA.Selenium.PageLoadStrategy.Normal;
             }
-            else if (PageLoadStrategy.ToLower() == nameof(OpenQA.Selenium.PageLoadStrategy.Eager).ToLower())
+            else if (PageLoadStrategy != null && PageLoadStrategy.ToLower() == nameof(OpenQA.Selenium.PageLoadStrategy.Eager).ToLower())
             {
                 options.PageLoadStrategy = OpenQA.Selenium.PageLoadStrategy.Eager;
             }
-            else if (PageLoadStrategy.ToLower() == nameof(OpenQA.Selenium.PageLoadStrategy.None).ToLower())
+            else if (PageLoadStrategy != null && PageLoadStrategy.ToLower() == nameof(OpenQA.Selenium.PageLoadStrategy.None).ToLower())
```

Comment:
Please check what PageLoadStrategy is coming null and if it is unavoidable, add single condition to handle null PageLoadStrategy .

Reply:
Done